packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -167,6 +174,39 @@ export default function CommentActionButtons({
     });
   }
 
+  const shouldShowFollow =
+    !useIsSpecialUser({ userId: comment?.author?.id }) &&
+    isLoggedIn &&
+    comment?.author &&
+    !isCompanion;
+
+  if (shouldShowFollow) {
+    const authorName = comment.author.name || `@${comment.author.username}`;
```

Comment:
Maybe let's add a ticket to make utility function for the `entityName` just because I've noticed we do different things :)

Reply:
https://dailydotdev.atlassian.net/browse/AS-594